LSP-pyright.sublime-settings

```diff
@@ -58,4 +54,10 @@
 		// Path to folder with a list of Virtual Environments.
 		"python.venvPath": "",
 	},
+	// Use a predefined setup from this plugin, valid values are:
+	// - "": An empty string will does nothing.
+	// - "st_dev": Suitable for people who are developing ST plugins.
+	//             `sys.path` from plugin_host will be added into "python.analysis.extraPaths"
+	//              so package dependecies will be resolved by the LSP server.
+	"use_predefined_setup": "",
```

Comment:
I'm trying to think of a better name for it. How about `dev_environment`? And then the value would be empty or `sublime_text`?

Reply:
Better than my original naming :+1: